compiler/one-cmds/one-codegen

```diff
@@ -113,6 +113,8 @@ def _parse_arg(parser):
     del argv[0]
     # split by '--'
     args = [list(y) for x, y in itertools.groupby(argv, lambda z: z == '--') if not x]
+    if len(args) == 0:
+        codegen_args = parser.parse_args(codegen_args)
```

Comment:
What is this change for? There is a similar change in one-infer and one-profile too.

Reply:
This is added for a technical reason. 

```diff
-    oneutils.parse_cfg(args, 'one-codegen')
+    oneutils.parse_cfg(args.config, 'one-codegen', args)
```

Before above diff, `args` can be a list or the obejct. But, after above diff, `args` always must be the returned object from argument parser. Which is why I changed `codegen_args`''s type from a list to the object when `args`'s length is 0.